app/build-js.sh

```diff
@@ -24,8 +24,8 @@ fi
 pushd ../node_modules/astro-sdk
 npm install
 grunt build_astro_client
-cp js/build/astro-client.js ../../app/scaffold-www/
 popd
+cp ../node_modules/astro-sdk/js/build/astro-client.js scaffold-www/
```

Comment:
I think we want this to be `$MYPATH/scaffold-www/` -- @jeremywiebe and I have a similar change sitting on a pending branch (navitron) that we made to make it work with the `npm link` stuff